Add clarity to Events in Sendgrid source and fixe a typo (#2259)
segmentio/segment-docs: src/connections/sources/catalog/cloud-apps/sendgrid/index.md

### Streaming

> The SendGrid source also has a streaming component which listens in real time for inbound webhooks from SendGrid's Event Notifications and batches the events to be uploaded on your next warehouse flush. ***These events are only ever appended to your warehouse.*** Note, if you don't use SendGrid's marketing features, this will be the only data that we receive from SendGrid. At present, there is no way to retrieve email event history from SendGrid, so you will only have access to data that we've collected after you successfully enable this component of the source destination.

Reviewer: ```suggestion
The SendGrid source also has a streaming component which listens in real time for inbound webhooks from SendGrid's Event Notifications and batches the events to be uploaded on your next warehouse flush. **These events only append to your warehouse.**

> note ""
> **NOTE:** If you don't use SendGrid's marketing features, this will be the only data that Segment receives from SendGrid. There isn't a way to retrieve email event history from SendGrid, so you will only have access to data that Segment collected after you successfully enable this component of the source destination.
```

**Object** collections are updated with each sync. These are pulled using our sync component.

> **Event** collections are append only, represent a user action or activity, and may be likened to fact tables in a traditional data warehouse. **Note:** Unlike traditional events captured by Segment, these events are not able to be forwarded to Destinations you've configured in your Segment workspace. Instead, they can only be synced to a supported data warehouse.

Reviewer: ```suggestion
**Event** collections are append only, represent a user action or activity, and may be likened to fact tables in a traditional data warehouse. **Note:** Unlike traditional events captured by Segment, you can't forward these events to Destinations you've configured in your Segment workspace. You can only sync these events to a supported warehouse.
```

If you're getting an "Invalid Credentials" error when setting up the SendGrid source, send a direct ping to the [SendGrid Marketing Campaigns API](https://sendgrid.com/docs/API_Reference/Web_API_v3/Marketing_Campaigns/campaigns.html) to test if you're using the correct credentials.

> Make sure you whitelist the Segment IP addresses on Sendgrid. [Contact our team](https://segment.com/help/contact/) for the list of IP addresses to whitelist.

Reviewer: ```suggestion
Make sure you whitelist the Segment IP addresses on Sendgrid. [Contact Segment](https://segment.com/help/contact/) for the list of IP addresses to whitelist.
```
